docs/modeling-guide.md

```diff
@@ -278,4 +278,67 @@ class FooContainer {
   baz: BazDefinition
   faz: FazDefinition
 }
+```
+
+### Additional information
+
+If needed, you can specify additional information on each type with the approariate JSDoc tag.
+Following you can find a list of the supported tags:
+
+#### `@since`
+
+Every API already has a `@since` tag, which describes when an API has been added.
+You can specify an additional `@since` tag for every parameter that has been added afterwards.
+If the tag is not defined, it's assumed that the parameter has been added with the API the first time
+
+```ts
+/**
+ * @since 7.10.0
+ */
+class FooRequest {
+  bar: string
+  /** @since 7.11.0 */
+  baz: string
+  faz: string
+}
+```
+
+#### `@description`
+
+A custom description of the property.
+
+```ts
+class Foo {
+  bar: string
+  /** @description You can baz! */
```

Comment:
Do we need an annotation for this ? This should be the jsdoc comment text:

```ts
/**
 * You can baz!
 */
```

And BTW we also should define the markup langage in descriptions. My suggestion is to use [CommonMark](https://commonmark.org/) that is implemented in all languages and that code generators can translate to whatever markup language is used in their respective documentation (some HTML with custom markers in JavaDocs, reStructuredText in Python, etc)

Reply:
Good suggestion, let me see what I can do about it.
Regarding the markup, I like the idea, but let's discuss it in a separate issue.